test/Engine/ProtoTest/DebugTests/BasicTests.cs

```diff
@@ -9374,6 +9375,36 @@ public void SteppingOverinline_Imperative_723_6()
             }
 
         }
+
+        [Test]
+        [Category("Debugger")]
+        public void TestStepInSimpleFunction()
+        {
+            string src =
+              @"def foo : int(a : int)
+                {
+                    return = a;
+                }
+                c1 = foo(10);";
+
+            fsr.PreStart(src, runnerConfig);
+            DebugRunner.VMState vms = fsr.Step();   // c1 = foo(10);
+
+            vms = fsr.Step();  // return = a;
+
+            Assert.AreEqual(3, vms.ExecutionCursor.StartInclusive.LineNo);
+            {
+                ExpressionInterpreterRunner watchRunner = new ExpressionInterpreterRunner(core);
+                ExecutionMirror mirror = watchRunner.Execute(@"a");
+                Obj objExecVal = mirror.GetWatchValue();
+                Assert.AreNotEqual(null, objExecVal);
+                Assert.AreEqual(10, (Int64)objExecVal.Payload);
+
+            }
+        }
+
+       
+
```

Comment:
I was debugging into a regression that involved step-in. There was no one simple step-in so i manually coded one. I thought since it helped in debugging, just add it in